pkg/import/import_resources.go

```diff
@@ -93,9 +93,7 @@ func Import(rd, definitionsFile io.Reader) ([]string, []string, error) {
 		for _, field := range variables {
 			if _, ok := v[field]; !ok {
 				return add, remove,
-					fmt.Errorf(
-						"error in resources definition %s: field '%s' doesn't exist in plan.", t, field,
-					)
+					fmt.Errorf("error in resources definition %s: field '%s' doesn't exist in plan", t, field)
 			}
 			id = append(id, fmt.Sprintf("%s", v[field]))
 		}
```

Comment:
On github, this whole block appears less indented then the enclosing code... So there is certainly a mix of tab and spaces in this file that should be fixed to improve reading the code.

Reply:
Strange, I didn't find any configuration  about the go  extension I'm using. The indentation is done by the IDE.

Reply:
@Esysc Note It is not the first time that I see this mix tabs spaces in your Go PRs.
Please take the time to understand how to configure properly your editor :-)
If using VSCode:
when a Go file is in the editor, at the bottom it should say: "Tab size: 4"
in the settings:
editor: detect indentation: checked
editor: insert spaces: checked (this will be overridden for Go files as long as Detect indentation is checked).

I think this is what you need.

maybe also editor: format on save: checked

Reply:
as long as the Go extension is configured correctly, and you do `code --add .` to add the directory containing `go.mod` to the workspace...

Reply:
Thanks for taking time for this exhaustive explanation.

Reply:
I've checked all these settings and it seems correct. Cannot find the root cause of this mess.